.env.staging

```
@@ -1,3 +1,4 @@
 NODE_ENV=production
 VUE_APP_ROOT_API=
 VUE_APP_HOST_ADDRESS=http://sustainability-kiosks.s3-website-us-west-2.amazonaws.com
+VUE_APP_FONT_PATH=/sustainability-kiosks/fonts/
```

```suggestion
VUE_APP_FONT_PATH=/fonts/
```
The [test deployment](
https://sustainability-kiosks.s3.us-west-2.amazonaws.com/index.html#/) still has a font loading error, after some testing it looks like it uses the same path as `.development` (I believe GH Pages is the outlier here, which is why `.production` needs the path it has).